lib/actions/ExtensionAction.js

```diff
@@ -194,10 +193,16 @@ class ExtensionAction {
         if (process.env['LOG_LEVEL'] === 'debug') logger.error(err)
         reject(new Error(`Error while downloading boilerplate: ${err.message}`))
       })
-      request(BOILERPLATE_URL).pipe(extractor)
+      request(this.getBoilerplateUrl()).pipe(extractor)
     })
   }
 
+  getBoilerplateUrl () {
+    if (process.env['BOILERPLATE_URL']) return process.env['BOILERPLATE_URL']
+    if (process.env['SDK_BETA']) return 'https://data.shopgate.com/connect/platform-sdk/beta.zip'
```

Comment:
`process.env['SDK_BETA']` is a string in this case.
You need to check explicit for the value.